### PR TITLE
Enabled ssl configuration

### PR DESCRIPTION
**Pull request by fadyham**
Enabled ssl encryption through Let's Encrypt
Added Prod and dev config files #15

### CODE REVIEW

File: src/Web.config [hunk 1]
```diff
@@ -15,6 +15,10 @@
     -->
   </appSettings>
   <system.webServer>
+    <staticContent>
+      <mimeMap fileExtension="." mimeType="text/plain" />
```
Comment (pmansour) [inline src/Web.config]: this doesn't look totally right to me. Would this allow anybody to access `https://hazzat.com/Configs/prod.json`?

File: src/app.ts [hunk 2]
```diff
@@ -19,6 +19,9 @@ const port = configuration.port;
 
 app.use("/", homeController.router);
 app.use("/seasons", seasonsController.router);
+
+// Allow Let's Encrypt to access challenge static content
+app.use("/.well-known/acme-challenge", express.static(__dirname + "/.well-known/acme-challenge"));
```
Comment (pmansour) [inline src/app.ts]: how does this file get populated?

File: src/Web.config (hunk 1 above)
Comment (fadyham) [reply src/Web.config]: No.  Just files with no extension.  Plus express will reject the /Config routes

File: src/app.ts (hunk 2 above)
Comment (fadyham) [reply src/app.ts]: There's an azure extension for Let's Encrypt.  It generates a challenge file in this directory whenever the SSL cert expires (every 3 months)

File: src/Web.config (hunk 1 above)
Comment (fadyham) [reply src/Web.config]: Looks like the .well-known directory has its own web.config file with the mime map.  This is not needed anymore.  Removed.

**Pull request merged**